src/appmixer/leadspicker/artifacts/openapi.json

```diff
@@ -3,7 +3,7 @@
     "info": {
         "title": "NinjaAPI",
         "version": "1.0.0",
-        "description": "",
+        "description": "Seamlessly capture leads from LinkedIn, optimize your email strategy with tailored outreach solution, and harness the power of email rotation for a guaranteed high delivery rate.",
```

Comment:
@vtalas I see that you have already created a patched JSON file with an updated description. Why do we need to update it here?

Reply:
that's the updated, original file is the openapi.orig.json